pyproject.toml

```
@@ -1,15 +1,86 @@
 [build-system]
 requires = [
-    "setuptools>=66.1.0; python_version>='3.12'",
-    "setuptools>=60.0.0; sys.platform=='win32'",
-    "setuptools<v60.0; python_version<'3.12' and sys.platform!='win32'",
+    "setuptools>=61.0.0",
+    "setuptools_scm<9",
     "cython>=0.28.0",
     "cmake",
     "numpy>=1.25.0; python_version>='3.9'",
     "oldest-supported-numpy; python_version<'3.9'",
 ]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "pyopenjtalk"
+dynamic = ["version"]
+description = "A python wrapper for OpenJTalk"
+readme = "README.md"
+requires-python = ">=3.7"
```

If you were to drop 3.7 it would look like this:
```suggestion
requires-python = ">=3.8"
```

---

Dropping py37 sounds good to me! I usually try to support as many versions as possible but numpy already removed the support for py37 quite a while ago. Thank you!